src/main/java/konkuk/thip/common/exception/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,68 @@
+package konkuk.thip.common.exception.handler;
+
+import konkuk.thip.common.dto.ErrorResponse;
+import konkuk.thip.common.exception.BusinessException;
+import konkuk.thip.common.exception.code.ErrorCode;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.MissingServletRequestParameterException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+@Slf4j
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
```

Comment:
P2: `NoHandlerFoundException.class` 예외처리도 추가하는 것은 어떨까요?
```suggestion
    //요청한 API가 없을때 예외처리
    @ExceptionHandler(NoHandlerFoundException.class)
    public ErrorResponse handleNoHandlerFoundException(NoHandlerFoundException e) {
        return new BaseErrorResponse(NOT_FOUND_API);
    }
```
이 예외처리를 해두면 프론트엔드분들과 협업할때도 그렇구, 포스트맨에서도 확인할때 정의하지않은 경로로 요청했을때나 오타같은 것들을 바로 확인할수 있어서 좋더라고요!

Reply:
저는 핸들링만추가하고 딱히 yml에 다른 설정같은건 하지않았어요!  
저도 Spring Boot 3.0 버전 사용했습니다!